src/wymeditor/core.js

```diff
@@ -944,6 +1022,44 @@ WYMeditor._computeBasePath = function (wymPath) {
     return wymPath.substr(0, lastSlashIndex + 1);
 };
 
+WYMeditor._getSkinUIOverrides = function (providedOptions, skin) {
```

Comment:
Some documentation would be useful for me to understand.

Also, I feel that this static method may be easier to understand if it also did the merging and not only provide the object for the merge.